src/pyff/utils.py

```diff
@@ -273,7 +273,13 @@ def load_url(url, enable_cache=True, timeout=60):
     if url.startswith('file://'):
         path = url[7:]
         if not os.path.exists(path):
-            raise IOError("file not found: %s" % path)
+            log.error("file not found: %s" % path)
+            return _Resource(result=None,
```

Comment:
I'd say keep the retry. When using pyff to download metadata files, especially the bigger ones, the retry with stand-off can increase the chance of a successful download.
It improves feedback to the user not to retry when it doesn't make sense to retry. Another case that could be added is a HTTP 404. Also adding this sped up the unit testing ;)